foundationdb/src/options.rs

```diff
@@ -147,7 +147,7 @@ impl NetworkOption {
             }
         }
     }
-    pub unsafe fn apply(&self) -> std::result::Result<(), error::FdbError> {
+    pub unsafe fn apply(&self) -> std::result::Result<(), error::Error> {
```

Comment:
I think we have a gap in our testing here... this should be regenerated and the result tested, is that happening now?